connectors/source.py

```diff
@@ -141,7 +141,11 @@ def _convert(self, value, field_type_):
         # list requires special type casting
         if cast_type is list:
             if isinstance(value, str):
-                return [item.strip() for item in value.split(",")] if value else []
+                return (
+                    [item.strip() for item in value.rstrip(",").split(",")]
+                    if value
+                    else []
+                )
```

Comment:
Would it make sense to move this function definition to utils module as we have comma-separated values in many connectors?

This way we can:
1. Re-use it
2. Have unit-tests for this logic outside of other tests.